src/GovUk.Education.ExploreEducationStatistics.Common/Services/PrivateBlobStorageService.cs

```diff
@@ -0,0 +1,23 @@
+#nullable enable
+using Azure.Storage.Blobs;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Services
+{
+    public class PrivateBlobStorageService : BlobStorageService
```

Comment:
I would go:

```
public class PrivateBlobStorageService : BlobStorageService, IPrivateBlobStorageService
```
 
here, and not have BlobStorageService implement IPrivateBlobStorageService and IPublicBlobStorageService anymore if I were you?

Reply:
Also, if you wanted to double down, stick all of this in the base constructor of BlobStorageService, and just have this constructor pass in the string "CoreStorage" to the base constructor, and the same for the public version too.

That way, you don't have to make the variables protected anymore.